test/unit/pdf/finance/invoice_generator_test.rb

```diff
@@ -43,10 +44,14 @@ class Pdf::Finance::InvoiceGeneratorTest < ActiveSupport::TestCase
     # ensure an image is present in the PDF
     assert_equal 1, content.scan(%r{/Type /XObject}).size
 
-    text = PDF::Inspector::Text.analyze(content)
+    strings = PDF::Inspector::Text.analyze(content).strings
     flat_items = items.flatten.reject(&:blank?).map(&:strip)
-    flat_items.each { |item| assert_includes text.strings, item }
-    assert_includes text.strings, "Chocolatte#{Prawn::Text::NBSP}" # prawn should not strip non-braking-spaces
+    flat_items.each { |item| assert_includes strings, item }
+    assert_includes strings, "Chocolatte#{Prawn::Text::NBSP}" # prawn should not strip non-braking-spaces
```

Comment:
As we're touching this line, we may as well fix the typo :) `non-breaking spaces`